pyproject.toml

```diff
@@ -35,6 +35,8 @@ dependencies = [
     "ansys-turbogrid-api>=0.4.dev1",
     "matplotlib>=3.5.1",
     "importlib-metadata>=4.0",
```

Comment:
These dependencies are only needed to run one or more specific examples; they aren't needed for general usage of the module. I think that there should be a better way to handle these; let's discuss offline.